content/en/docs/zero-code/java/spring-boot-starter/annotations.md

```diff
@@ -1,57 +1,94 @@
 ---
 title: Annotations
-weight: 60
+weight: 50
+cSpell:ignore: proxys
 ---
 
 For most users, the out-of-the-box instrumentation is completely sufficient and
 nothing more has to be done. Sometimes, however, users wish to create
 [spans](/docs/concepts/signals/traces/#spans) for their own custom code without
 doing too much code change.
```

Comment:
I know you didn't change this as part of this PR, so maybe this could be addressed in a separate PR, but I think the readability could be improved for this sentence.
```suggestion
needing to make many code changes.
```